OneSignalExample/Assets/OneSignal/CHANGELOG.md

```diff
@@ -8,8 +8,10 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ### Changed
 - `InstallEdm4uStep` now imports version [1.2.177](https://github.com/googlesamples/unity-jar-resolver/releases/tag/v1.2.177) of [EDM4U](https://github.com/googlesamples/unity-jar-resolver)
 - Updated included Android SDK to [5.0.1](https://github.com/OneSignal/OneSignal-Android-SDK/releases/tag/5.0.1)
+- Updated included iOS SDK to [5.0.2](https://github.com/OneSignal/OneSignal-iOS-SDK/releases/tag/5.0.2)
 ### Fixed
 - Sending VSAttribution data from the editor
+- iOS notifications clicked event to fire when app is opened from clicking on a notification
```

Comment:
I would include the wording "cold start" in there. Something like "iOS notifications clicked event firing if the app was cold started from clicking a notification"